README.md

```diff
@@ -9,12 +9,12 @@ building
 linux
 -----
 
-Requires [vagrant](https://www.vagrantup.com/)
+Requires [docker](https://www.docker.com/)
 
-+ Change directories to the target arch.
-+ `vagrant up && vagrant ssh && cd /vagrant_data` to pop yourself in a controlled image back at this repo's root
++ Change directories to the root of this repository.
++ `docker run -v "$(pwd):/pantsbuild-binaries" -it --entrypoint /bin/bash python:2.7.13-wheezy && cd /pantsbuild-binaries` to pop yourself in a controlled image back at this repo's root
```

Comment:
almost all docker commands should include `--rm` to clean up after, or else you'll end up leaking disk space. during wheelbuilder dev, I exceeded 60GB in Docker file content and ran out of disk space (with no apparent easy way to fix other than another docker incantation).

also, I wonder if it would make sense to just provide this as a simple shell script and then instruct users to run e.g. `./bin/docker_shell` vs the full command which could also do things like check for `docker` in $PATH and other user friendly validations.